src/KOKKOS/atom_vec_kokkos.cpp

```diff
@@ -16,8 +16,11 @@
 #include "atom_kokkos.h"
 #include "atom_masks.h"
 #include "comm_kokkos.h"
+#include "error.h"
 #include "domain.h"
 
+#define DELTA 10
```

Comment:
@akohlmey this is the key line.

Reply:
thanks. can we then please add a comment to the `roundup()` function explaining this?

we should also replace the `#define DELTA` with a `const int DELTA` inside the `roundup()` function, same for `atom_vec.cpp`. The compiler will ultimately create the same code, but we gain more readable code and type checking.

Reply:
>we should also replace the #define DELTA with a const int DELTA inside the roundup() function, same for atom_vec.cpp

This type of #define is used everywhere in vanilla LAMMPS too. I suppose the idea for including it up at the top of the file was so you only had to change it in one place?

Reply:
Now that lammps support `c++11`, why don't you replace these macros with `constexpr` e.g. `static constexpr std::size_t kDelta = 10;` at the top?

Reply:
> > we should also replace the #define DELTA with a const int DELTA inside the roundup() function, same for atom_vec.cpp
> 
> This type of #define is used everywhere in vanilla LAMMPS too. I suppose the idea for including it up at the top of the file was so you only had to change it in one place?

that would make sense if it actually *would* be used in multiple places throughout the entire code and then consistently so by putting the define into a commonly used header.

but in this case it is quite confusing or else I would not have asked the question starting this discussion in the first place.

Reply:
Understood, I just like to keep the Kokkos version as similar to vanilla as possible, helps when porting changes  to Kokkos.

Reply:
at any rate, it would be sufficient for me to merge this pull request to have the comment to the roundup() function(s) added explaining why the KOKKOS version needs to be re-implement.

Reply:
> Understood, I just like to keep the Kokkos version as similar to vanilla as possible, helps when porting changes to Kokkos.

It was my understanding to change both functions/files. But don't worry about it. I'll deal with that elsewhere.

Reply:
OK I'll put in some comments

Reply:
@akohlmey I added the comments.